py/tests/test_tasks.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+from unittest.mock import patch
+import ctl.tasks
+from app import app
+
+
+class TasksViewTestCase(unittest.TestCase):
+    @patch('ctl.tasks.flask')
+    @patch('dao.utils.mysql')
+    def test_connect_to_database(self, mysql_connector, flask):
```

Comment:
we should rename this function when merging. it has been given when i were trying to understand what to do with the mySQL.

Reply:
Ah, ok, thanks - I'll take care of this... You are right, this is a tricky question, whether we should mock mysql or dao layer itself... Anyway it wouldn't be hard for me to change later after you showed the way :+1: